config/Dockerfiles/tests.d/libvirt/06_libvirt-multi-node-network

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -xe
+
+# Verify basic provisioning for all supplied providers
+# distros.exclude: centos7 fedora30
+# providers.include: libvirt
+
+## NOTE: This is a temporary test to run libvirt tests until we fix
+## the networking race condition. Tests will only run on centos7.
+
+set -o pipefail
+
+DISTRO=${1}
+PROVIDER=${2}
+
+TEST_NAME="${DISTRO}/${PROVIDER}/06_libvirt-multi-node-network"
+DESCRIPTION="Test ${PROVIDER} provider with multi-node-network provisioning"
+
+echo "Test Name: ${TEST_NAME}"
+echo "${DESCRIPTION}"
+
+TARGET="libvirt-multi-node-net"
+
+pushd docs/source/examples/workspaces/${PROVIDER}
+
+function clean_up {
+    set +e
+    linchpin -w . -v destroy ${TARGET}
+    # remove output file used to gen uhash if it exists
+    rm resources/ -rf || true
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+linchpin -w . -v up ${TARGET} 
+pushd resources/
+IP_NUM=$(cat ${TARGET}.output|grep -o ip|wc -l)
+if [ ${IP_NUM} -eq '6']; then
```

Comment:
I think your logic is backwards here.  Shouldn't it be `if [ ${IP_NUM} -ne '6' ]`?

Reply:
Yes I see your logic, but when you run the code, when the IP_NUM == 6, it returns false I know it's weird